components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/internal/streaming/SubscriptionHelper.java

```diff
@@ -99,50 +99,51 @@ public SubscriptionHelper(final SalesforceComponent component) throws Salesforce
 
         this.listenerMap = new ConcurrentHashMap<>();
 
-        restartBackoff = new AtomicLong(0);
-        backoffIncrement = component.getConfig().getBackoffIncrement();
-        maxBackoff = component.getConfig().getMaxBackoff();
+        this.restartBackoff = new AtomicLong(0);
+        this.backoffIncrement = component.getConfig().getBackoffIncrement();
+        this.maxBackoff = component.getConfig().getMaxBackoff();
     }
 
     @Override
     protected void doStart() throws Exception {
 
         // create CometD client
-        this.client = createClient(component);
+        this.client = createClient(this.component);
 
         // reset all error conditions
-        handshakeError = null;
-        handshakeException = null;
-        connectError = null;
-        connectException = null;
+        this.handshakeError = null;
+        this.handshakeException = null;
+        this.connectError = null;
+        this.connectException = null;
 
         // listener for handshake error or exception
-        if (handshakeListener == null) {
+        if (this.handshakeListener == null) {
             // first start
-            handshakeListener = new ClientSessionChannel.MessageListener() {
-                public void onMessage(ClientSessionChannel channel, Message message) {
+            this.handshakeListener = new ClientSessionChannel.MessageListener() {
+                @Override
+                public void onMessage(final ClientSessionChannel channel, final Message message) {
                     LOG.debug("[CHANNEL:META_HANDSHAKE]: {}", message);
 
                     if (!message.isSuccessful()) {
                         LOG.warn("Handshake failure: {}", message);
-                        handshakeError = (String)message.get(ERROR_FIELD);
-                        handshakeException = getFailure(message);
+                        SubscriptionHelper.this.handshakeError = (String)message.get(ERROR_FIELD);
```

Comment:
What is the reason to add `SubscriptionHelper.this` for these properties? IMHO it makes the code a bit chaotic, same goes for `this`.

Reply:
Sorry is my mistake... (default configuration of my IDE). I fix this.

Reply:
I commited a more correct patch